model/src/main/java/com/epam/esm/entity/GiftCertificate.java

```diff
@@ -1,34 +1,68 @@
 package com.epam.esm.entity;
 
-import com.fasterxml.jackson.annotation.JsonFormat;
-
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
-import java.util.List;
+import java.util.Set;
 
 /**
  * Entity of a gift certificate.
  *
  * @author Dzmitry Ahinski
  */
+@Entity(name = "gift_certificates")
+@Table(name = "gift_certificates")
 public class GiftCertificate {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "certificate_id")
     private long id;
+    @Column(name = "certificate_name")
     private String name;
     private String description;
     private BigDecimal price;
     private int duration;
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
     private LocalDateTime createDate;
-    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
     private LocalDateTime lastUpdateDate;
-    private List<Tag> tags;
+
+    @OneToMany(mappedBy = "certificate")
+    private Set<Order> orders;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+            name = "gift_certificates_has_tags",
+            joinColumns = @JoinColumn(name = "gift_certificates_id_fk"),
+            inverseJoinColumns = @JoinColumn(name = "tags_id_fk")
+    )
+    private Set<Tag> tags;
 
     public GiftCertificate() {
 
```

Comment:
Extra line

Reply:
Can not understand why they are extra, please explain

Reply:
Extra line between constructor and field?